scripts/exglobal_atmos_analysis.sh

```diff
@@ -678,14 +678,17 @@ if [ ${DOHYBVAR} = "YES" ]; then
       SETUP="niter(1)=50,niter(2)=150,niter_no_qc(1)=25,niter_no_qc(2)=0,thin4d=.true.,ens_nstarthr=3,l4densvar=${l4densvar},lwrite4danl=${lwrite4danl},${SETUP}"
       JCOPTS="ljc4tlevs=.true.,${JCOPTS}"
       STRONGOPTS="tlnmc_option=3,${STRONGOPTS}"
-      OBSQC="c_varqc=0.04,airs_cads=.false.,cris_cads=.false.,iasi_cads=.false.,${OBSQC}"
+      OBSQC="c_varqc=0.04,${OBSQC}"
    fi
 fi
 
 if [ ${DONST} = "YES" ]; then
    NST="nstinfo=${NSTINFO},fac_dtl=${FAC_DTL},fac_tsl=${FAC_TSL},zsea1=${ZSEA1},zsea2=${ZSEA2},${NST}"
 fi
 
+# Infrared cloud and aerosol detection software (CADS) on (.true.) - off (.false.) flags 
+OBSQC="airs_cads=${AIRS_CADS},cris_cads=${CRIS_CADS},iasi_cads=${IASI_CADS},${OBSQC}"
```

Comment:
Since the `*_cads` variables are not inside an if-block, this `OBSQC` line will be executed every time `exglobal_atmos_analysis.sh` is run.  

Given this we could remove lines 689-690 and simply add `airs_cads=${AIRS_CADS},cris_cads=${CRIS_CADS},iasi_cads=${IASI_CADS}` to the `OBSQC` namelist on lines 751 to 756.  For example,
```
&OBSQC
  dfact=0.75,dfact1=3.0,noiqc=.true.,oberrflg=.false.,c_varqc=0.02,
  use_poq7=.true.,qc_noirjaco3_pole=.true.,vqc=.false.,nvqc=.true.,
  aircraft_t_bc=.true.,biaspredt=1.0e5,upd_aircraft=.true.,cleanup_tail=.true.,
  tcp_width=70.0,tcp_ermax=7.35, airs_cads=${AIRS_CADS},
  cris_cads=${CRIS_CADS}, iasi_cads=${IASI_CADS},
  ${OBSQC}